### DISCUSSION TITLE
Show last time a particular object type was detected

### DISCUSSION THREAD

**Discussion by ronisaacson** [blakeblackshear/frigate#13669] [Ask A Question]
With the new UI, is there a way to see the last time a certain object was detected, the way it worked in the old UI? For example, if I want to know the last time a bear was detected no matter how long ago it was, without searching day by day, how would I do that?

I don't generally look at my cameras unless I have a reason to. One day last year, I opened frigate to check the live view, and saw that a bear had been detected several days earlier. And yes it was an actual bear! I would never have seen it except for how the old UI would show the most recent instance of each object. It would also show false positives for bears sometimes, and this made it easy to spot and unlabel them.

Also what's with the type icons? I see in the new UI that it shows a little picture of a car, person, etc. when that type of object is in the clip. I don't think I've had a bear picked up since upgrading, but if I did... would it just show the clip with no icon? Would Bear become a selection in the filter dropdown?

**Comment by hawkeye217** [comment]
A new view has been implemented for the next version of Frigate that will allow you to do this and more.

Many objects that Frigate detects have icons associated with them, including `bear`. And yes, `bear` would become a selection in the filter dropdown.
